test/UnitTests/UnitTests.csproj

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <RootNamespace>System.Data.Entity</RootNamespace>
     <AssemblyName>EntityFramework.UnitTests</AssemblyName>
-    <TargetFramework>$(NetFrameworkToolCurrent)</TargetFramework>
+    <TargetFrameworks>$(NetFrameworkToolCurrent);$(DefaultNetCoreTargetFramework)</TargetFrameworks>
```

Comment:
IIUC this means we weren't running previously these unit tests on .NET Core, only on .NET Framework; the change appears to make us run **some** tests on .NET Core, even if quite a few are excluded because of various dependencies on .NET Framework.

@AndriySvyryd can you give this a look and confirm?

Reply:
Correct, it seemed like it "couldn't hurt" to add the extra safety of running the many tests that do work correctly in the current test code under .NET Core.

That's why I wanted to ask y'all about your preference for the unit test strategy and csproj. I could definitely tweak it so that only the single `LinqExpressionNormalizerTests.cs` file is also run under .NET Core. Just wanted to get your opinion/preferences.